packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -273,6 +298,21 @@ function Cell(props) {
     cellContent = children;
   }
 
+  let columnHighlight = {};
+  if (columnHighlightColor === ColumnHighlightColor.GREEN) {
+    columnHighlight = {
+      'column-highlight-green': true,
+      'first-highlight-green': rowId === firstRowId,
+      'last-highlight-green': rowId === lastRowId,
+    };
+  } else if (columnHighlightColor === ColumnHighlightColor.ORANGE) {
+    columnHighlight = {
+      'column-highlight': true,
+      'first-highlight': rowId === firstRowId,
+      'last-highlight': rowId === lastRowId,
+    };
+  }
```

Comment:
You should just have a classes to determine if the color is orange or green.  The first and last highlight should not depend upon that.  I would suggest creating the class from the enumeration constant value like many of our other implementations.

Reply:
Refactored in https://github.com/cerner/terra-framework/pull/1985/commits/fce4b663063a694bddaeaf64d6bd4ce240c1020f